stable/grafana/templates/NOTES.txt

```diff
@@ -20,9 +20,9 @@
    NOTE: It may take a few minutes for the LoadBalancer IP to be available.
         You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "grafana.fullname" . }}'
      export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "grafana.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-     http://$SERVICE_IP:{{ .Values.service.port -}}
```

Comment:
This may be a dumb question: where does the grafana.ini file come from?

Reply:
...and how does it differ from .Values?

Reply:
Let's discuss offline.